### DISCUSSION TITLE
BUG Arcade Shortcut not working in tadpole versions - FIX Workaround

### DISCUSSION THREAD

**Discussion by bn40400** [EricGoldsteinNz/tadpole#50] [General]
***Original Discussion/ Question***

Hello, I've used the latest tadpole tool version 0.4.2, and I've tried to change my 4 shortcut slots under the Mame (Arcade) section and none of the slots appear to work. I'll select them, get a slight flicker and loading screen, then immediately goes back to the home screen. I was able to successfully change the Famicom shortcuts with no issue, they work well. I just can't get the arcade ones to work. Any help and suggestions would be appreciated - thanks! 

***Edit***

I figured it out, arcade shortcuts in the Resources/xfgle.hgp are created incorrectly. They point to the .zfb files in the ARCADE folder rather than the corresponding .zip file in the ARCADE/bin folder. I was able to manually edit the invalid entries using notepad to the correct ones and the shortcuts now work. Hope this helps.

**Comment by EricGoldsteinNz** [comment]
Closing as this is a duplicate of [#49](https://github.com/EricGoldsteinNz/tadpole/discussions/49)
